js/documents.js

```diff
@@ -838,7 +848,7 @@ var documentsMain = {
 		}
 
 		documentsMain.ready = true;
-		if (parent && parent.OCA.Files.App.fileList.reload) {
+		if (parent && documentsMain.getFileList() !== null) {
 			parent.OCA.Files.App.fileList.reload();
```

Comment:
Must not all uses of parent.OCA.Files.App.fileList be replaced by documentsMain.getFileList()?

Reply:
Good catch, thanks :+1: